RealmSwift/Map.swift

```diff
@@ -814,7 +814,7 @@ extension Map: Sequence {
 
 extension Map {
     /// An adaptor for Map which makes it a sequence of `(key: Key, value: Value)` instead of a sequence of `SingleMapEntry`.
-    public struct KeyValueSequence<Key: _MapKey, Value: RealmCollectionValue>: Sequence {
```

Comment:
Duplicating the generic arguments here was incorrect and now produces a warning.